packages/web3.js-plugin-fee-estimation/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../../tsconfig.json",
```

Comment:
Unrelated to PR. We should move the tsconfig.json to a package sometime. It's a tiny antipattern to require files outside of a package without depending on them. I haven't gotten around to doing this yet though because it's super super low priority though

Reply:
I agree, I could fix this since I have a limited amount of work on my plate while ramping up

Reply:
Just opened an issue for this today might be worth giving it a week or 2 to see if an outside contributor looking for a first issue wants to pick it up https://github.com/ethereum-optimism/optimism/issues/6629